EIC-EA/EIC-community-D8
# EICNET-520: Update blocks Page Banner and CTA Tiles

### Improvements

- Update field_cta_link to a field type "Typed link" in both Paragraph CTA Tiles and block Page Banner;
- Add css classes to cta tiles links based on the selected link type;
- Update exported "Homepage - Page banner" block;
- Update exported "Stay up to date" block;
- Add missing fragment cache tags to fact and figures;
- Add missing paragraphs cache tags to cta_tiles;

### Fixed issues

- Added misssing block_content cache tags to renderable array. This issue is temporarly solved this way until it gets fixed in **context** contrib module.

### Tests

- Install platform using deploy script;
- Check if the "Homepage - Page banner" block matches the mockups;
- Check if the "Stay up to date" matches the mockups;
- Edit the "Stay up to date" block content and try to change the one of the CTA Tiles link;
- Check if the link style is updated on the homepage;
